README.md

```diff
@@ -48,6 +39,39 @@ Check out [p5js.org](https://p5js.org) for lots more! Here are some quick-links
 * [Libraries](https://p5js.org/libraries): Extend p5 functionality to interact with HTML, manipulate sound, and more!
 * [The Coding Train p5.js Tutorials](https://thecodingtrain.com/beginners/p5js/): A huge trove of tutorials created by Dan Shiffman and friends
 
+
+## Issues
+
+If you have found a bug in the p5.js library, you can file it here under the [“issues” tab](https://github.com/processing/p5.js/issues). You can also request new features here. A set of templates for reporting issues and requesting features are provided to assist you (and us!). p5.js is maintained mostly by volunteers, so we thank you for your patience as we try to address your issues as soon as we can. Please post in the correct repository if you can:
+
+* p5.js general: [https://github.com/processing/p5.js/issues](https://github.com/processing/p5.js/issues)
+* p5.sound: [https://github.com/processing/p5.js-sound/issues](https://github.com/processing/p5.js-sound/issues)
+* p5.js website: [https://github.com/processing/p5.js-website/issues](https://github.com/processing/p5.js-website/issues)
+* p5.js web editor: [https://github.com/processing/p5.js-web-editor/issues](https://github.com/processing/p5.js-web-editor/issues)
+
+
+## Get Started for Developers
+
+If you want to work/contribute to P5's🌸 codebase as a developer, either directly for improving P5 or for improving it's sub projects like [Friendly Error Systems](https://github.com/processing/p5.js/blob/main/contributor_docs/friendly_error_system.md), you can follow the following steps directly :
+
+1. Create a fork of P5 repo.
+2. Clone your locally created fork.
+3. Add upstream using the following command : 
+   `git remote add upstream https://github.com/processing/p5.js`
+   [More information on Configuring a remote repository for a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork)
+4. Make sure your machine has [NodeJs](https://nodejs.org/en/download) installed, check it with the        following command : 
+   `node -v`
+5. Run : `npm ci`
+6. Create a git branch of the `main` branch having a descriptive branch name using :
+   `git checkout -b [branch_name]`
+7. As you start making changes to the codebase, frequently run : 
+    `npm test`
+  (it takes time, but it ensures that existing behaviours are not being broken)
+8. Once it is done, you can commit the changes and create a [Pull Request](https://p5js.org/contributor-docs/#/./contributor_guidelines?id=pull-requests). 
```

Comment:
Instead of detailing the steps here, would it be possible to just link to the relevant section in the contributor docs? We had problems before where there are multiple places documenting the same thing (on README, contributor docs, wiki, website, inline comments etc) but gradually go out of sync with each other and become hard to maintain.

Reply:
thnx for commenting on this @limzykenneth! if u think it's best to move this elsewhere just let us know where u'd prefer we put it, but our thinking is that this sort of high-level info for setting up a dev environment is fundamental for anyone interested in contributing to the repository && is the sort of thing contributors on other FLOSS projects might expect to find in a repo's README. It took both @Ayush23Dash + i (who are not new to FLOSS but are new to p5.js) a bit of work to figure out what the basic dev setup was supposed to look like && i fear that moving this elsewhere could make it harder to find for future folks forking the repo with the hopes of contributing. that said, i'm new to this project, so u all might have had convos about this in the past && might have some other reasons why u might want to omit the dev setup steps from the README. so i leave that to ur discretion.

Reply:
+1 to that, also for the problem you mentioned @limzykenneth , we could maybe add a note to update the documentation at both places, whenever it is being done. Saying this because we believe it actually will solve one actionable problem!!

Reply:
I guess one thing that gives me concern is the amount of detail here. If comparing with other open source repos, for example, the fork, clone (setting upstream remote is also quite detailed), and node.js installation steps won't be included and it would usually start at the `npm install` step, although many open source projects actually don't include that information in the README anymore (eg. lodash, Vue.js, three.js all don't have instructions but just links to contribution documentation). 

Commit and submitting pull request on the other hand, if to be mentioned also possibly missing some details that may be good to have especially for someone new to open source contributions. Not to mention most PR for p5.js should have issues opened for discussions first.

I think if the problem is that the dev setup is too difficult to figure out, we should aim to simplify it instead of adding more documentation (we have a lot already, and I'm guilty of making quite a few of them :stuck_out_tongue:). My aim for most projects would be a clone, then `npm install` (or `npm ci` if desired), then `npm test` should be the bare minimum setup.

PS. "p5.js" is the project name, while it is fine to refer to the project as p5, it shouldn't be capitalized (P5) and for very old (maybe irrelevant) historical reason, p5 was used to refer to Processing so p5.js would be a clearer indication of the project itself.

Reply:
The current README file is already quite long, and we can reorganize it. I agree with @limzykenneth that we can keep the dev setup documentation at one place like the existing contributor doc, and potentially link the dev setup in the main README. 
Meanwhile, we can aim to simplify the dev setup process for the next release or so.

Reply:
hey yawwwl, thnx for sharing ur thoughts on all this. i hear ya. the steps @Ayush23Dash put together could probably be a little less verbose, the README is already pretty long, && there are already lots of docs (which is great, but can also be overwhelming for new contributors). so maybe we close this PR && think about putting this information elsewhere (maybe the contributor guidelines as suggested?)

while efforts to make the build setup process simpler can be hlepful, i don't think that's the issue this PR is aiming to solve. i think Ayush is trying to answer the question many new code contributors might have which is "how do i setup a local dev environment?" or rather "how do you--p5.js--want me to setup the environment?" (which is a prerequisite for working on any code related issues). again, it's not that the dev setup is too difficult to figure out (we were able to figure it out by poking around in the package.json && inferring what/how that all worked && how i should test my changes after some experimentation), it's just that different FLOSS projects have different setups && not having it explicitly stated had us wondering, "are we doing it right?" (even after having figured it out). but again, maybe that's info that belongs in the contributor guidelines? just under "Get Started" or maybe under "Source code contribution"?

Reply:
So based on the above discussion and as per discussion with @almchung && @nbriz in today's meet, I wanted to ask that can we move the above steps(in fewer words) to the Contributor docs and just put a link in this main Readme to the particular section in the Contributor Doc's where the precise information to build the project would be present, so that new contributors(especially who are beginners in the coding/FOSS world)  can directly get to the required information to build and locally set up the project!?

Reply:
Yes, that sounds great, @Ayush23Dash. Feel free to make the changes. @limzykenneth wrote the most recent contributor doc in 2022 as part of Season of Docs program, so please let me and @limzykenneth know if you need clarification.

Reply:
@Ayush23Dash Yes, a short summarized version of the minimal steps required to build would be fine to add. You can add this to the top of the contributor guidelines document and link to that specific section from the Readme. You might want to consider linking to the web version of it instead too: https://p5js.org/contributor-docs/#/contributor_guidelines

Reply:
Hi @limzykenneth , I have implemented the changes with a few corrections to the existing Contributor Guidelines! It would be great if you could review it now!
Thanks!